test/host.js

```diff
@@ -21,17 +21,14 @@ describe('host', () => {
 		].forEach((src) => {
 			it(`should throw invalid origin "${src}"`, () => {
 				expect(() => {
-					var host = new Host('id', src);
+					var host = new Host(() => { return null; }, src);
```

Comment:
Not a big deal, but I think this can be simplified to: `new Host(() => null, src);`. There's another case further down too.